flutter/omise/android/local.properties

```diff
@@ -0,0 +1,2 @@
+sdk.dir=/Users/shunsuke.h/Library/Android/sdk
```

Comment:
@shunsuke-h-opn seems like this should not be committed

Reply:
@danfowler 
Oh, sorry, I will remove it.

Reply:
@danfowler 
done
https://github.com/omise/examples/pull/34/commits/534d7853192136e5a54ba0cc89eb0b8859eca2a9